backend/typescript/rest/reviewRoutes.ts

```diff
@@ -80,18 +80,24 @@ reviewRouter.get(
   },
 );
 
-reviewRouter.delete(
-  "/:id",
-  isAuthorizedByRole(new Set(["Admin"])),
-  async (req, res) => {
+reviewRouter.put("/:id", reviewRequestDtoValidator, async (req, res) => {
```

Comment:
can we add the `isAuthorizedByRole` and `isAuthorizedByUserId` middleware here?

Reply:
I must've missed it in the merge conflict resolution. I'll add them back